### ISSUE TITLE
A puzzle about the model in chapter 2.

### ISSUE THREAD

**Issue by xzhws** [SherlockLiao/pytorch-beginner#18]
The model in chapter 2 should be multilayer perceptron instead of logistic regression, since which can only be used in two classfication problem. How about changing the model to softmax which is available for multi-classification problem? thanks.

**Comment by L1aoXingyu** [comment]
You are right, theoretically logistic regression deal with two-class classification problem and I will think about change this example to a two-class problem.

**Comment by JHCM120665** [comment]
ok
